shell/packages/sandstorm-permissions/permissions.js

```diff
@@ -357,10 +370,11 @@ downstreamTokens = function(root) {
     grainId = root.grain._id;
   }
 
-  var grain = Grains.findOne(grainId);
+  var grain = db.collections.grains.findOne(grainId);
```

Comment:
In theory we should be moving away from using db.collections and instead adding methods to db itself covering our needs, so that we can clearly see what kinds of queries we do by reviewing the SandstormDb interface. In this case a getGrain() method makes a lot of sense.